.vscode/settings.json

```diff
@@ -1,3 +1,12 @@
 {
     "cmake.sourceDirectory": "${workspaceFolder}/src",
+    "lean.executablePath": "bin/lean",
+    "files.associations": {
+      "*.lean": "lean",
+      "*.hl": "isabelle-ml",
+      "*.sdr": "glsl",
```

Comment:
Can you please revert this change?

Reply:
oops, not sure how that got in there. --assume-unchanged is always finicky...